pkg/kubelet/images/image_gc_manager.go

```diff
@@ -457,10 +538,23 @@ func (im *realImageGCManager) imagesInEvictionOrder(ctx context.Context, freeTim
 			continue
 
 		}
-		images = append(images, evictionInfo{
-			id:          image,
-			imageRecord: *record,
-		})
+		if !isRuntimeClassInImageCriAPIEnabled {
+			images = append(images, evictionInfo{
+				id:          image,
+				imageRecord: *record,
+			})
+		} else {
+			imageTuples := strings.Split(image, ",")
```

Comment:
suggestion: Maybe make this a function like `getImageTuple` to make it  clear that is what is happening here.

Reply:
not a huge deal but you didn't use the helper here

Reply:
@haircommander I haven't pushed changes to address James's review comments still. Are you also suggesting the same as James is or am I missing something?

Reply:
I thought that's what https://github.com/kubernetes/kubernetes/pull/121456/files#diff-887859ee44bc4f73de8b1a5e92feab5bda8b12fa5f452859d46037d6eaefc3adR549 was

Reply:
@haircommander Hit the push command by mistake but hit ctrl-C. Didn't realize that it had already gone through. Aplogies for the confusion! The PR is updated now with all comments addressed. Thanks!